src/chocolatey/infrastructure.app/runners/GenericRunner.cs

```diff
@@ -111,188 +111,188 @@ private ICommand find_command(ChocolateyConfiguration config, Container containe
+        public void Run(ChocolateyConfiguration config, Container container, bool isConsole, Action<ICommand> parseArgs)
         {
             var tasks = container.GetAllInstances<ITask>();
             foreach (var task in tasks)
             {
-                task.initialize();
+                task.Initialize();
             }
 
-            fail_when_license_is_missing_or_invalid_if_requested(config);
-            SecurityProtocol.set_protocol(config, provideWarning: true);
-            EventManager.publish(new PreRunMessage(config));
+            FailOnMissingOrInvalidLicenseIfFeatureSet(config);
+            SecurityProtocol.SetProtocol(provideWarning: true);
+            EventManager.Publish(new PreRunMessage(config));
 
             try
             {
-                var command = find_command(config, container, isConsole, parseArgs);
+                var command = FindCommand(config, container, isConsole, parseArgs);
                 if (command != null)
                 {
                     if (config.Noop)
                     {
                         if (config.RegularOutput)
                         {
-                            this.Log().Info("_ {0}:{1} - Noop Mode _".format_with(ApplicationParameters.Name, command.GetType().Name));
+                            this.Log().Info("_ {0}:{1} - Noop Mode _".FormatWith(ApplicationParameters.Name, command.GetType().Name));
                         }
 
-                        command.noop(config);
+                        command.DryRun(config);
                     }
                     else
                     {
-                        this.Log().Debug("_ {0}:{1} - Normal Run Mode _".format_with(ApplicationParameters.Name, command.GetType().Name));
-                        command.run(config);
+                        this.Log().Debug("_ {0}:{1} - Normal Run Mode _".FormatWith(ApplicationParameters.Name, command.GetType().Name));
+                        command.Run(config);
                     }
                 }
             }
             finally
             {
-                EventManager.publish(new PostRunMessage(config));
+                EventManager.Publish(new PostRunMessage(config));
 
-                foreach (var task in tasks.or_empty_list_if_null())
+                foreach (var task in tasks.OrEmpty())
                 {
-                    task.shutdown();
+                    task.Shutdown();
                 }
 
-                remove_nuget_cache(container, config);
+                RemoveNugetCache(container, config);
             }
         }
 
         /// <summary>
         /// if there is a NuGetScratch cache found, kill it with fire
         /// </summary>
         /// <param name="container">The container.</param>
-        private void remove_nuget_cache(Container container)
+        private void RemoveNugetCache(Container container)

```

Comment:
To be correct, this should really be `RemoveNuGetCache`. This could be a required change elsewhere as well.

Reply:
I don't disagree, but I will point out that we have `NugetService` among several other things in this codebase that use this casing. :/

Reply:
Fixed this for now, but if we wanna fix this properly, there's a bunch more work and deprecations in our future :/

![image](https://user-images.githubusercontent.com/32407840/229806129-430bca80-37ee-464f-b7d4-d4784e8a0cdd.png)

Reply:
Yeah, I thought that this would be a much bigger change.  Let's shelf that particular change for now :-D